semgrep-core/src/parsing/tree_sitter/Parse_swift_tree_sitter.ml

```diff
@@ -1350,9 +1352,9 @@ and map_directly_assignable_expression (env : env)
   match x with
   | `Simple_id x -> map_simple_identifier env x
   | `Navi_exp x -> map_navigation_expression env x
-  | `Call_exp x -> map_call_expression env x
+  | `Call_exp x -> map_call_expression env x |> todo env
```

Comment:
question: why is `todo` here? It looks like you implemented `map_call_expression` as part of this PR